Gems/ROS2/Code/Source/ROS2Module.cpp

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Some of these files are missing copyright helpers
1.Gems/ROS2/Code/Source/ROS2EditorSystemComponent.h
2. Gems/ROS2/Code/Source/ROS2Module.cpp
3. Gems/ROS2/Code/Source/ROS2EditorSystemComponent.cpp
4. Gems/ROS2/Code/Source/ROS2EditorModule.cpp
5. Gems/ROS2/Code/CMakeLists.txt
6. Gems/ROS2/CMakeLists.txt

Reply:
I will add them. Would you say it also makes sense to also add them to the Gem template?

Reply:
yes, though it is off by default:
```
  -kl, --keep-license-text
                        Should license in the template files text be kept in
                        the instantiation, default is False, so will not keep
                        license text by default. License text is defined as
                        all lines of text starting on a line with
                        {BEGIN_LICENSE} and ending line {END_LICENSE}.
```

Reply:
Then that sounds like we have a bug with some of the files then applying the copyright, when they shouldn't.

Reply:
It was my omission - I was not aware about the flag previously, included license manually for some but not all files.